new version

xavo95 opened the pull request:
i let you here the pull request if you want to merge

cj123 commented:
the difficult thing here is that your commit messages aren't particularly helpful and I have no way of knowing what's changed. Could you provide a summary here?

xavo95 commented:
Tomorrow i will give a detailed information. But resuming from you version, i have added a fix for memory issues(provided by iNeal), the port discovery(you could add or delete ports modifying index.php and statusupdate.txt) and today I added a temperature check

Enviat des del meu xPhone

El 25/07/2015, a les 17:25, Callum Jones <notifications@github.com> va escriure:
> the difficult thing here is that your commit messages aren't particularly helpful and I have no way of knowing what's changed. Could you provide a summary here?
> 
> —
> Reply to this email directly or view it on GitHub.
>
